replica/replica.go

```diff
@@ -179,6 +180,10 @@ func construct(readonly bool, size, sectorSize int64, dir, head string, backingF
 		}
 	}
 
+	if err := r.isExtentSupported(); err != nil {
+		return nil, err
```

Comment:
Use something like: `return nil, errors.Wrap(err, "file extent is unsupported")`. So we can see what the error is clearly in the log.

Other than this, LGTM.